inc/osvr/PluginKit/EyeTrackerInterface.h

```diff
@@ -0,0 +1,115 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
```

Comment:
I'd suggest removing this file for now, since there aren't matching ones for the other interfaces. The only reason I added one for Imaging is because there's some manipulation required to the C++ OpenCV data structure before we can pass it through the C API.